src/Microsoft.SqlTools.ServiceLayer/Security/UserData.cs

```diff
@@ -793,7 +799,8 @@ public bool IsDefaultLanguageSupported
             get
             {
                 //Default Language was not supported before Denali.
-                return SqlMgmtUtils.IsSql11OrLater(this.context.Server.ConnectionContext.ServerVersion);
+                return SqlMgmtUtils.IsSql11OrLater(this.context.Server.ConnectionContext.ServerVersion)
+                    && this.context.Server.ServerType != DatabaseEngineType.SqlAzureDatabase;
```

Comment:
Does `DatabaseEngineType.SqlAzureDatabase` contain MI? default language should be supported with ALTER USER in MI 

https://learn.microsoft.com/en-us/sql/t-sql/statements/alter-user-transact-sql?view=azuresqldb-mi-current&preserve-view=true

Reply:
Let's take the MI case post-March since I want to get these changes in today.  I'll test and open a bug to track that if necessary.